scripts/import_standard_ebooks.py

```diff
@@ -0,0 +1,166 @@
+from __future__ import annotations
+
+import json
+import requests
+import sys
+import time
+
+import os.path as path
+
+import feedparser
+
+from openlibrary.core.imports import Batch
+
+sys.path.insert(0, path.abspath(path.join(path.sep, 'openlibrary')))
+from openlibrary.config import load_config  # noqa: E402
+load_config(
+    path.abspath(path.join(
+        path.sep, 'olsystem', 'etc', 'openlibrary.yml')))
+from infogami import config  # noqa: E402,F401
+
+FEED_URL = 'https://standardebooks.org/opds/all'
+LAST_UPDATED_TIME = './standard_ebooks_last_updated.txt'
+
+
+def get_feed():
+    """Fetches and returns Standard Ebook's feed."""
+    r = requests.get(FEED_URL)
+    return feedparser.parse(r.text)
+
+
+def map_data(entry) -> dict:
+    """Maps Standard Ebooks feed entry to an Open Library import object."""
+    std_ebooks_id = entry.id.replace('https://standardebooks.org/ebooks/', '')
+    return {
+        "title": entry.title,
+        "source_records": [f"standardebooks:{std_ebooks_id}"],
+        "publishers": [entry.publisher],
+        "publish_date": entry.dc_issued[0:4],
+        "authors": [{"name": author.name} for author in entry.authors],
+        "description": entry.content[0].value,
+        "subjects": [tag.term for tag in entry.tags],
```

Comment:
I'm not seeing `tags` anywhere in https://standardebooks.org/opds/all ; does this get `<category scheme="http://purl.org/dc/terms/LCSH" term="Errors and blunders, Literary"/>`?

Reply:
Yes, `tags` is a dictionary of categories for an entry: https://feedparser.readthedocs.io/en/latest/reference-entry-tags.html